CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -0,0 +1,55 @@
+- Add OVS table name as label for 'ovs_flow_count' Prometheus metrics. ([#4893](https://github.com/antrea-io/antrea/pull/4893), [@cr7258])
+- Add WireGuard tunnel mode for Multi-cluster to establish encrypted connection between member clusters. ([#4606](https://github.com/antrea-io/antrea/pull/4606), [@hjiajing])
+- Watch EndpointSlice API by default for exported Service's endpoint change in Multi-cluster. ([#4895](https://github.com/antrea-io/antrea/pull/4895), [@luolanzone])
+
+### Changed
+
+- Deprecate Antrea Octant Plugin. ([#4825](https://github.com/antrea-io/antrea/pull/4825), [@antoninbas])
+- Update Open vSwitch to 2.17.6. ([#4959](https://github.com/antrea-io/antrea/pull/4959), [@tnqn])
+- Update Windows OVS version to 2.16.7. ([#4705](https://github.com/antrea-io/antrea/pull/4705), [@XinShuYang])
+- Stop serving v1alpha2 version of the ClusterGroup CRD. ([#4812](https://github.com/antrea-io/antrea/pull/4812), [@antoninbas])
+- Change to cache Flow modification message in Antrea Agent to improve Agent memory usage. ([#4495](https://github.com/antrea-io/antrea/pull/4495), [@wenyingd])
+- Replace PacketIn/Controller with PacketIn2/Controller2 to improve packetin handler. ([#4768](https://github.com/antrea-io/antrea/pull/4768), [@GraysonWu])
+- Change to look up Pods by name instead of IP address to fetch labels in Flow Aggregator, to avoid obtaining incorrect Pods when Pod turnover quickly. ([#4942](https://github.com/antrea-io/antrea/pull/4942), [@dreamtalen])
+
+### Fixed
+
+- Use LOCAL instead of CONTROLLER as the in_port of packet-out message to fix Windows agent crash issue. ([#4992](https://github.com/antrea-io/antrea/pull/4992), [@tnqn])
+- Fix the Antrea Agent crash issue on Windows by running modules that rely on Services after AntreaProxy is ready. ([#4946](https://github.com/antrea-io/antrea/pull/4946), [@tnqn])
+- Fix a bug that a deleted policy is still enforced when a new policy with the same name exists. ([#4986](https://github.com/antrea-io/antrea/pull/4986), [@tnqn])
+- Make FQDN NetworkPolicy work for upper case DNS. ([#4934](https://github.com/antrea-io/antrea/pull/4934), [@GraysonWu])
+- Fix a bug that K8s Networkpolicy audit logging not working for Service access. ([#4780](https://github.com/antrea-io/antrea/pull/4780), [@qiyueyao])
+- Fix Service not being updated correctly when stickyMaxAgeSeconds or InternalTrafficPolicy is updated. ([#4845](https://github.com/antrea-io/antrea/pull/4845), [@tnqn])
+- Fix EndpointSlice API availablility check to resolve the issue that AntreaProxy always falls back to the Endpoints API when EndpointSlice is enabled. ([#4852](https://github.com/antrea-io/antrea/pull/4852), [@tnqn])
+- Prevent headless Service from updating Service CIDR to avoid overwriting the default route of host network unexpectedly. ([#5008](https://github.com/antrea-io/antrea/pull/5008), [@hongliangl])
```

Comment:
Probably add "In Antrea Agent Service CIDR discovery," to be clear.

headless Service -> headless Services

the discovered Service CIDR